Solutions/SharpArch.Domain/DomainModel/BaseObject.cs

```diff
@@ -138,12 +121,12 @@ public virtual bool HasSameObjectSignatureAs(BaseObject compareTo)
                 return base.Equals(compareTo);
             }
 
-            for (int index = 0; index < signatureProperties.Length; index++)
+            for (var index = 0; index < signatureProperties.Length; index++)
             {
                 var property = signatureProperties[index];
```

Comment:
I thought we had decided to only use vars when the initializer has the type in https://github.com/sharparchitecture/Sharp-Architecture/issues/81 but doesn't seem we reached a consensus in the end. :)